packages/aws-cdk-lib/aws-cloudfront-origins/lib/s3-bucket-origin.ts

```diff
@@ -211,9 +211,9 @@ export abstract class S3BucketOrigin extends cloudfront.OriginBase {
           principals: [this.originAccessIdentity.grantPrincipal],
         }));
         if (!result.statementAdded) {
-          Annotations.of(scope).addWarningV2('@aws-cdk/aws-cloudfront-origins:updateImportedBucketPolicy',
-            'Cannot update bucket policy of an imported bucket. You may need to update the policy manually instead.\n' +
-            'See the "Using pre-existing S3 buckets" section of module\'s README for more info.');
+          Annotations.of(scope).addWarningV2('@aws-cdk/aws-cloudfront-origins:updateImportedBucketPolicyOai',
+            'Cannot update bucket policy of an imported bucket. You will need to update the policy manually instead.\n' +
+            'See the "Setting up OAI with imported S3 buckets (legacy)" section of module\'s README for more info.');
```

Comment:
There are 2 unit tests need to be updated for these 2 warning message updates.

https://github.com/samson-keung/aws-cdk/blob/06b471a6b572d5ad42e39fb6f592d0bae902457f/packages/aws-cdk-lib/aws-cloudfront-origins/test/s3-bucket-origin.test.ts

Line 699 and Line 1151

Reply:
good catch, will update